lib/plugins/create/templates/azure-nodejs-event-hub/README.md

```diff
@@ -0,0 +1,133 @@
+# Azure Functions - Quick Start
```

Comment:
This file seems to be copied over from the docs and might need some overhaul in terms of wording and links to the actual docs.